### DISCUSSION TITLE
User insertion upon login

### DISCUSSION THREAD

**Discussion by ChungusPrime** [JakyeRU/Larascord#29] [Q&A]
Hi there,
I was wondering which function gets called when inserting the user into the database for the first time?

I would assume it is the createOrUpdateUser() function at the bottom of DiscordController.php, however I placed a die/dump before this function and it doesn't trigger, any help please?

**Comment by ValentinNeaga** [comment]
Hello Aaranihlus,

Yes, the `createOrUpdateUser` from `DiscordController.php` (line 106) is called whenever a new user needs to be created or updated.

https://github.com/JakyeRU/Larascord/blob/6af6fa9cd93bae87fd7950dbe63ac8a8c826c0ff/src/Http/Controllers/DiscordController.php#L106
